src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerCreateDraftOrder.java

```diff
@@ -55,6 +55,7 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
         final CaseDetails<CaseData, State> beforeDetails
     ) {
 
+
         return AboutToStartOrSubmitResponse.<CaseData, State>builder()
```

Comment:
Is this space change needed?

Reply:
no  but when i format the code  it makes spaces between the lines. this is only format the code